src/recipe/Baked.Recipe.Service.Application/CodingStyle/RichTransient/InitializeUsingIdParameterConvention.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Baked.Business;
+using Baked.Domain.Model;
+using Baked.RestApi.Configuration;
+using Baked.RestApi.Model;
+using Humanizer;
+
+namespace Baked.CodingStyle.RichTransient;
+
+public class InitializeUsingIdParameterConvention(DomainModel _domain)
+    : IApiModelConvention<ActionModelContext>
+{
+    public void Apply(ActionModelContext context)
+    {
+        if (!context.Controller.MappedType.TryGetMembers(out var members)) { return; }
+        if (!members.Methods.Having<InitializerAttribute>().Any()) { return; }
+        if (!members.Has<LocatableAttribute>()) { return; }
+        if (context.Controller.MappedType.TryGetQueryType(_domain, out var _)) { return; }
```

Comment:
this creates a dependency to orm feature. don't use orm or data access or related coding style extensions.